attributes/tomcat.rb

```diff
@@ -135,3 +135,5 @@
 default['alfresco']['activiti_tomcat_instance']['xmx_ratio'] = 0.1
 activiti_memory = "#{(node['memory']['total'].to_i * node['alfresco']['activiti_tomcat_instance']['xmx_ratio'] ).floor / 1024}m"
 default['alfresco']['activiti_tomcat_instance']['java_options']['xmx_memory'] = "-Xmx#{activiti_memory}"
+
+default['supervisor']['tomcat']['user'] = "tomcat"
```

Comment:
We shouldnt have an extra attribute for this. You should use the already existent node[tomcat][user] which is set in _tomcat-attributes.rb